nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -316,7 +361,7 @@ public DocumentRef getTopLevelCommentAncestor(CoreSession s, DocumentRef documen
     /**
      * Gets or creates the 'Comments' folder, this folder will be under the document being commented and it contains the
      * whole comments of the first document that we comment.
-     * 
+     *
```

Comment:
IDE automatically removes trailing spaces. Since there are no cleanup commits in backports I can either leave it as it is, or put it back in a text editor, but the problem will happen again and again in next commits.